lib/ansible/modules/cloud/amazon/inspector_run.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/python
+# Copyright (c) 2018 Dennis Conrad for Sainsbury's
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: inspector_run
+short_description: Start, Stop, and Delete Amazon Inspector Assessment Runs
+description: Starts, stops, and deletes Amazon Inspector Assessment Runs.
+version_added: "2.6"
+author: "Dennis Conrad (@dennisconrad)"
+options:
+  name:
+    description:
+      - The name of the assessment run.  Required unless C(state=started), in
+        which case it is optional.
+      - If not specified and C(state=started), a name based on the assessment
+        template name, time, and a random string will be generated.
+  state:
+    description:
+      - The state of the assessment run.
+    choices:
+      - started
```

Comment:
As part of convention for Ansible module UX, we like to have a `present` option that's the default. For example, on the AWS EC2 modules `present` is equivalent to `ensure the instance exists, and if you must create it then do so. If it exists and is stopped, leave it stopped`. For Inspector I'd expect a `present` to start a job from the specified template, unless one exists already and is stopped or running.